client/src/index.js

```diff
@@ -1,39 +1,35 @@
-import React from "react";
+import React, { useState } from "react";
 import ReactDOM from "react-dom";
 import "./index.css";
 import Home from "./components/Home";
 import "semantic-ui-css/semantic.min.css";
 import About from "./components/About";
 import Status from "./components/Status";
-import { BrowserRouter as Router, Route, Link } from "react-router-dom";
+import { NavLink, BrowserRouter as Router, Route } from "react-router-dom";
+import { Menu, Container, Image, Input } from "semantic-ui-react";
 
 const Routes = () => {
   return (
     <Router>
-      <div>
-        <ul>
-          <li>
-            <Link className="nav-link" to="/">
-              Home
-            </Link>
-          </li>
-          <li>
-            <Link className="nav-link" to="/about">
-              About
-            </Link>
-          </li>
-          <li>
-            <Link className="nav-link" to="/status">
-              Status
-            </Link>
-          </li>
-        </ul>
+      <Menu inverted>
+        <Container>
+          <Menu.Item header>
```

Comment:
`header` styles the menu item **bold** so that it looks more like a heading. Since there is no text here, it does not do anything so let's remove it.